docs/tutorials/connecting-to-the-internet-via-usb.md

```diff
@@ -137,447 +108,223 @@ If you are using an older version, please upgrade.
-            Acer driver, which doesn't work properly with ev3dev. Click *Update
-            driver...* to change the driver.
-
-            {% include screenshot.html source="/images/screenshots/windows/10/rndis/usb-ethernt-rndis-gadget-properties-acer.png" %}
-
-        14. Select *Browse my computer for driver software*.
-
-            {% include screenshot.html source="/images/screenshots/windows/10/rndis/update-driver-software-usb-ethernet-rndis-gadget.png" %}
-
-        15. Then *Let me pick from a list of device drivers on my computer*.
-
-            {% include screenshot.html source="/images/screenshots/windows/10/rndis/update-driver-software-usb-ethernet-rndis-gadget-2.png" %}
-
-        16. Select *Have Disk*.
-
-            {% include screenshot.html source="/images/screenshots/windows/10/rndis/update-driver-software-usb-ethernet-rndis-gadget-3.png" %}
-
-        17. Click *Browse...*.
-
-            {% include screenshot.html source="/images/screenshots/windows/10/rndis/install-from-disk-1.png" %}
-
-        18. Navigate to the location that you saved the `linux.inf` file earlier.
-            Select it and click *Open*.
-
-            {% include screenshot.html source="/images/screenshots/windows/10/rndis/locate-file-linux-inf.png" %}
-
-        19. Click *OK*.
-
-            {% include screenshot.html source="/images/screenshots/windows/10/rndis/install-from-disk-2.png" %}
-
-        20. Pick *Remote NDIS Compatible Device* from the list, then click *Next*.
-
-            {% include screenshot.html source="/images/screenshots/windows/10/rndis/update-driver-software-usb-ethernet-rndis-gadget-4.png" %}
-
-        21. Windows doesn't like the fact that our .inf file is not signed. Don't
-            worry, it just installs the Microsoft RNDIS driver (which is signed).
-            Nothing really dangerous here. Click *Install this driver software anyway*.
+*   {: tab="Windows"}<span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
+    These instructions were written using Window 10, but should work Windows
+    7 and 8 as well.
+    {:class="alert alert-info"}
 
-            {% include screenshot.html source="/images/screenshots/windows/10/rndis/windows-security-install-anyway.png" %}
+    1.  Before connecting the USB cable, go ahead and open *Devices and Printers*
+        on your computer. It is in the *Control Panel* under *Hardware and Sound*.
+        (Or just type *Devices and Printers* in the start menu.)
 
-        21. When it is done installing, click *Close*.
+        {% include screenshot.html source="/images/windows/10/devices-and-printers.png" %}
 
-            {% include screenshot.html source="/images/screenshots/windows/10/rndis/update-driver-software-usb-ethernet-rndis-gadget-5.png" %}
+    2.  Connect your EV3 to your Windows computer using the USB cable that came
+        with the EV3. You should see an unknown device pop up right away.
```

Comment:
Not sure. Either it will continue using the existing driver or the new driver will load automatically. Either way, it should "just work".